### ISSUE TITLE
Unify coordinates

### ISSUE THREAD

**Issue by mschilli87** [rajewsky-lab/polyA#90]
I cannot explain why I needed to put `+ 1` in a91d707.
I would have expected `- 1` instead (see https://github.com/rajewsky-lab/polyA/pull/85#issue-166049397).

We should unify coordinates and distance calculations _before_ adding the minus strand (#36).

**Comment by mschilli87** [comment]
@nukappa: Earlier you said you agreed that it should be `- 1` and you wouldd have an idea of why I had to use `+ 1` instead.

Can you explain it so I can have a look?
Or are you sure enough/is it easier for you to just fix it yourself?

So please let me know what you want me to do/try or assign yourself.

**Comment by nukappa** [comment]
yeah, let's put it in details tomorrow. hard to explain cause i wrote a paper with "conventions" and what is what, we should do it in person
